QUICK_START.md

```diff
@@ -0,0 +1,174 @@
+# Quick Start Guide
+
+Get vTeam running locally in **under 5 minutes**! 🚀
+
+## Prerequisites
+
+Install these tools (one-time setup):
+
+### macOS
+```bash
+brew install minikube kubectl podman
+```
+
+### Linux
+```bash
+# Install kubectl
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+chmod +x kubectl
+sudo mv kubectl /usr/local/bin/
+
+# Install minikube
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+sudo install minikube-linux-amd64 /usr/local/bin/minikube
+
+# Install podman
+sudo apt install podman  # Ubuntu/Debian
+# or
+sudo dnf install podman  # Fedora/RHEL
+```
+
+## Start vTeam
```

Comment:
If you installed podman in the previous step, you need to do:

```
podman machine init
```

and also if you haven't started podman since your last shutdown you need to do:

```
podman machine start
```